### PR TITLE
DSR-2028: Updated README.md

### PR DESCRIPTION
**Pull request by dpazj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,82 @@
...
+`sudo apt-get install lua5.3`
 
+## Run locally
 
-## Run  
+To run the simulator locally build the binary and then run the command:
 
   `./build/release/ble-sim/ble-sim`
+
+To run the simulator with a lua script, pass use the --script option with the path to the lua script as its argument:
```
Comment (Tylerg2471) [inline README.md]: need to remove either "pass" or "use"

**Review by Tylerg2471 — CHANGES_REQUESTED** [review]
One minor comment

File: README.md (hunk 1 above)
Comment (dpazj) [reply README.md]: done

**Review by Tylerg2471 — APPROVED** [review]
LGTM

**Pull request merged**